apps/dashr-svm/app.R

```diff
@@ -0,0 +1,514 @@
+library(dashR)
```

Comment:
:pencil2: `dashR` to `dash` (given recent changes to the package, and in preparation for deployment)